pauthabielf64/pauthabielf64.rst

```diff
@@ -777,13 +777,15 @@ Static Linking
 ==============
 
 The static linker cannot create signed pointers, just as it cannot run
-constructors for static variables, but the C-runtime that runs before
-main can. The static linker must communicate the details of how to
+constructors for static variables, but the runtime that runs before
```

Comment:
How about "but the startup code that runs before the program (``main`` in C/C++) can?

Avoids "runtime that runs" which sounds a bit redundant.

There isn't really a good definition that I can point to. https://en.wikipedia.org/wiki/Entry_point uses the term library startup code and main as the start of the user program.